.github/pr_path_labeler.yml

```diff
@@ -0,0 +1,46 @@
+# Labeler triggered by .github/workflows/labels.yml
+
+github:
+- any: [".github/**/*", "!.github/workflows"]
```

Comment:
Do we have a good reason for giving GitHub actions its own label but not the other CI services? Doesn't the generic `CI` label suffice?

Reply:
No strong reason on my side. There is a `github` tag right now. Nothing for the rest.

Reply:
I think `github` is fine, if it's also about things like PR and issue templates, repo organization, etc.